packages/plugins/plugin-inbox/src/components/Mailbox/Mailbox.tsx

```diff
@@ -59,42 +59,48 @@ const renderMessageCell = (message: DataType.Message, now: Date, _isCurrent?: bo
       data-inbox-action="current-message"
       data-message-id="${id}"
     >
-      <p class="message__abstract__heading">
-        <span class="message__abstract__from">${from}</span>
-        <span class="message__abstract__date">${date}</span>
-      </p>
-      <p class="message__abstract__body">${subject}</p>
-    ${
-      message.properties?.tags
-        ? `<div class="message__tag-row">
-            ${message.properties.tags.map((tag: Tag) => `<div class="dx-tag message__tag-row__item" data-label="${tag.label}" data-hue=${tag.hue}>${tag?.label}</div>`).join('')}
-          </div>`
-        : ''
-    }
-    </button>`;
+      <div class="message__abstract__heading">
+        <div class="message__abstract__from">${from}</div>
+        <div class="message__abstract__date">${date}</div>
+      </div>
```

Comment:
Why change these from semantic components to nonsemantic `div`s? There is no cause for this change and it reduces the accessibility of these components.